cypress/integration/organizations.spec.js

```diff
@@ -10,13 +10,13 @@ describe("create organization", () => {
         cy.get(loginModel.email).type(dataOrg.user.email)
         cy.get(loginModel.password).type(dataOrg.user.pass)
         cy.get(loginModel.logInBtn).click();
-       
-      });
 
-     //positive
-     it("Add new org", () => {
-      
-        cy.get(organizations.addNewOrgBtnDropdown).eq(1).click({force: true});
+    });
+
+    //positive
+    it("Add new org", () => {
+
+        cy.get(organizations.addNewOrgBtnDropdown).eq(1).click({ force: true });
```

Comment:
Why do you need `force: true` here? Why use `eq(1)` as well?

Reply:
It is necessary because we have 3 el. for that select, also here is dropdown btn and needs to be "force: true". Otherwise, I get the issue: 
https://prnt.sc/-4eYXpp7np-3